yb-voyager/cmd/miginfo.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type MigInfo struct {
+	SourceDBType    string
+	SourceDBName    string
+	SourceDBSchema  string
+	SourceDBVersion string
```

Comment:
As suggested by Sanyam, also pick Oracle specific fields that are included in the schema report.

Reply:
They are not yet included in the Schema report. I have added them to the miginfo variable. I'll add them to the report as well if you want.